python/cudf/dataframe/series.py

```diff
@@ -1224,6 +1224,15 @@ def mean_var(self, ddof=1):
     def sum_of_squares(self, dtype=None):
         return self._column.sum_of_squares(dtype=dtype)
 
+    def round(self, decimals=0):
+        """
+        """
+        if self.empty:
+            return self
+
+        rounded = cudautils.apply_round(self.data.to_gpu_array(), decimals)
```

Comment:
It is suggested to use `self.data.mem` instead of `self.data.to_gpu_array()` when we need a view:

https://github.com/rapidsai/cudf/issues/1606

While it appears both return a view currently, it might be more future-proof to use `mem` when a view is required.

Reply:
`.to_gpu_array()` only returns a view when there's no nulls in the input and this function doesn't care about nulls so it should absolutely use `self.data.mem`

Reply:
👍

Reply:
Updated to make sure it's always a view with `self.data.mem`